tests/test_models.py

```
@@ -0,0 +1,35 @@
+@pytest.mark.django_db
+def test_patient_create():
+    p = Patient.objects.create(
+        full_name="Ali Test", 
+        primary_doctor_id=uuid.UUID("11111111-1111-1111-1111-111111111111")
+    )
+    assert p.id is not None
+    assert p.full_name == "Ali Test"
+
+
+@pytest.mark.django_db
+def test_encounter_link():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_encounter_link`

Add return type annotation: `None`

(ANN201)